packages/pigeon/platform_tests/test_plugin/windows/pigeon/core_tests.gen.cpp

```diff
@@ -545,10 +545,9 @@ const flutter::StandardMessageCodec& HostIntegrationCoreApi::GetCodec() {
 void HostIntegrationCoreApi::SetUp(flutter::BinaryMessenger* binary_messenger,
                                    HostIntegrationCoreApi* api) {
   {
-    auto channel =
-        std::make_unique<flutter::BasicMessageChannel<flutter::EncodableValue>>(
-            binary_messenger, "dev.flutter.pigeon.HostIntegrationCoreApi.noop",
-            &GetCodec());
+    auto channel = std::make_unique<flutter::BasicMessageChannel<>>(
```

Comment:
This churn is from a minor drive-by streamlining I did: the `*Channel` classes default to `flutter::EncodableValue` as the template type, so `flutter::BasicMessageChannel<>` is equivalent to `flutter::BasicMessageChannel<flutter::EncodableValue>>`, but shorter.